hazelcast/src/main/java/com/hazelcast/internal/management/operation/UpdateTcpIpMemberListOperation.java

```
@@ -0,0 +1,92 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.operation;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.TcpIpConfig;
+import com.hazelcast.internal.dynamicconfig.ClusterWideConfigurationService;
+import com.hazelcast.internal.dynamicconfig.ConfigurationService;
+import com.hazelcast.internal.management.ManagementDataSerializerHook;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.impl.NodeEngineImpl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Operation to update the member list in the tcp-ip join config at runtime.
+ */
+public class UpdateTcpIpMemberListOperation extends AbstractManagementOperation  {
+    private List<String> members;
+
+    public UpdateTcpIpMemberListOperation() {
+    }
+
+    public UpdateTcpIpMemberListOperation(List<String> members) {
+        this.members = members;
+    }
+
+    @Override
+    public void run() throws Exception {
+        Config config = ((NodeEngineImpl) getNodeEngine()).getNode().getConfig();
+        Object activeNetworkConfig;
+        TcpIpConfig tcpIpConfig;
+        if (config.getAdvancedNetworkConfig().isEnabled()) {
+            activeNetworkConfig = config.getAdvancedNetworkConfig();
+            tcpIpConfig = config.getAdvancedNetworkConfig().getJoin().getTcpIpConfig();
+        } else {
+            activeNetworkConfig = config.getNetworkConfig();
+            tcpIpConfig = config.getNetworkConfig().getJoin().getTcpIpConfig();
+        }
+        if (tcpIpConfig.isEnabled()) {
+            tcpIpConfig.setMembers(members);
+            ConfigurationService configurationService
+                    = getNodeEngine().getService(ClusterWideConfigurationService.SERVICE_NAME);
+            configurationService.persist(activeNetworkConfig);
```

If I get this correctly
- this statement will throw an exception if config persistence (an EE feature) is not enabled.
- resolving issues similar to #18661 requires the ability to manually update the members list

so for OS & EE-without-config-persistence users this issue will remain. Could we wrap this in a try-catch to allow for a transient (in-memory-only) member list update?

---

I catched and ignored the exceptions thrown in `configurationService.persist()` call